src/js/components/Form/stories/Components.js

```diff
@@ -71,6 +72,7 @@ const Example = () => (
             min={15}
             max={75}
           />
+          <FormField label="File" name="file" component={FileInput} />
```

Comment:
The Reset behavior of the Form is not working well with the FileInput component.

Reply:
should be fixed